jetty-infinispan/infinispan-remote/pom.xml

```diff
@@ -40,5 +108,23 @@
          <artifactId>infinispan-common</artifactId>
          <version>${project.version}</version>
     </dependency>
+    <dependency>
+         <groupId>org.infinispan</groupId>
+         <artifactId>infinispan-client-hotrod</artifactId>
+         <version>${infinispan.version}</version>
+         <scope>provided</scope>
+    </dependency>
+    <dependency>
+         <groupId>org.infinispan</groupId>
+         <artifactId>infinispan-remote-query-client</artifactId>
+         <version>${infinispan.version}</version>
+         <scope>provided</scope>
+    </dependency>
+    <dependency>
+         <groupId>org.infinispan.protostream</groupId>
+         <artifactId>protostream</artifactId>
+         <version>4.2.2.Final</version>
```

Comment:
maybe the last 4.2.3.Final?

Reply:
I'm using the same version as infinispan depends on. It's fragile that I have to have a dependency on protostream at all (as infinispan will eventually upgrade, and we'll have to remember to update the pom), but I can't see what else I can do.